playbooks/roles/curator/templates/action.yml.j2

```diff
@@ -3,6 +3,8 @@ actions:
   1:
     action: delete_indices
     description: "Delete indices older than {{ curator_delete_indices_days }} days."
+    options:
+      ignore_empty_list: True
```

Comment:
Should we add `continue_if_exception: True` in case there's an error with one of the indices we can continue and delete the rest instead of waiting for somebody to fix the issue?
Also, I'm not sure which indices are we deleting here but I'm think that's probably old indices that hold edx courses metadata for search. I believe those indices can grow quite big, should we also add a way to manage the timeout in case there's an issue in the future?
`timeout_override: <seconds>`

Reply:
@xirdneh These are the logs for almost everything from our infrastructure. We only keep them for 6 months because it's a massive amount of data and we don't have enough storage to keep that around for more. Also, it doesn't make much sense to keep logs for more time than that.
Does `continue_if_exception` logs the error? Because if it doesn't it might lead to the logs machine to fail silently due to out of disk space errors, so that's why I'd rather not include it.

Reply:
@giovannicimolin I would expect that weather the exception gets logged or not has to do with the logging configuration, but I'm not entirely sure about it.
Given the size of the indices I think it would be a good idea to see if we can add these two config values since it's recommended for very large indices.